web_src/js/features/repo-projects.js

```diff
@@ -193,20 +193,6 @@ export function initRepoProject() {
     const url = $(this).data('url');
     createNewColumn(url, columnTitle, projectColorInput);
   });
-
-  $('.new-project-column').on('input keyup', (e) => {
```

Comment:
Why `input` event is also removed?

Reply:
It's not working. The `input` event is to enable/disable the `confirm` button. But the `confirm` button is not disabled in the first place. We don't use this kind of enable/disable attribute in other buttons either. So I removed it.

Reply:
The frontend and backend are both introduced in the PR that implements the `Project` feature. So It has some personal code style. As I said in the description, It would be great if this part of frontend code is refactored to align other POST forms that we used in Gitea.